packages/x-charts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "7.22.0",
+  "version": "7.22.1",
```

Comment:
Agreed, doesn't look like anything changed between `7.22.0` and now. 🤷

Reply:
I'd rather do an empty release and keep all the version numbers in sync, wdyt?

Reply:
Feel like we are having this debate every two weeks :laughing: 
Our decision in the general meeting a few month ago was to just follow what Lerna updates, to at least be consistent everytime.

Reply:
Last time when I was releasing we "agreed" to release all packages regardless
https://github.com/mui/mui-x/pull/14910#discussion_r1795107844
Script was updated to reflect that

Reply:
Ok, the problem/question in this case is clear.
We have two cases:
1. We currently do not release when Lerna doesn't detect any changes and there were no specific changes in the package;
2. We release, but mention no changes in the CHANGELOG when no specific changes happened on the package, but Lerna detects changes (usually because of dependencies bumping)

This difference in approach is somewhat confusing and leads to unstable results. 🤔 

This is why I slightly prefer the more manual approach—releasing only the packages with changes to it and skipping in case of dependency bumping.

WDYT about the compromise of not listing the package in the changelog if it doesn't have any changes in the 2nd case? 🤔

Reply:
> Last time when I was releasing we "agreed" to release all packages regardless

That's not what I understand from this discussion.
Lerna can release packages without PR tight to it (if you have a dep that was updated for example).

Reply:
> WDYT about the compromise of not listing the package in the changelog if it doesn't have any changes in the 2nd case? 🤔

I think we should list the package in the changelog if it was released, even if there are no notable changes.
And whether we should release the package or not – determined by Lerna.

Reply:
Gotcha, I'm fine with that as well.
But this approach is still creating debates, that is why we can possibly improve it. 👍 🤔

Reply:
>> Last time when I was releasing we "agreed" to release all packages regardless

> That's not what I understand from this discussion.

Here we go again 😅
I've opened https://github.com/mui/mui-x/pull/15219 to clarify this in the release instructions.
I'm gonna need each team's approval on it to make sure we're on the same page.
If there's anything to discuss or clarify further – leave a comment in https://github.com/mui/mui-x/pull/15219